docs/getting-started/installation.md

```diff
@@ -0,0 +1,51 @@
+# How to install propsd #
+
+[Releases of propsd][releases] include both source tarballs and Debian
+packages. Debian and Ubuntu based Linux distributions can use the pre-built
+packages. Other operating systems should install propsd from source.
+
+## Installing from the Debian package ##
+
+Propsd runs on the 4.4.x LTS version of Node.js, so follow the [instructions
+for installing Node.js on Debian based systems][node-debian].
+
+Download a pre-built Debian package of propsd from [the releases
+page][releases] and save it. These instructions assume you've saved the package
+to `/tmp/propsd.deb`.
+
+Use `dpkg` to install propsd.
+
+~~~bash
+dpkg -i /tmp/propsd.deb
+~~~
+
+Propsd is installed into `/opt/propsd`.
+
+## Installing from source ##
+
+Propsd runs on the 4.4.x LTS version of Node.js, so follow the [instructions
+for installing Node.js][node-source].
+
+Download a tarball of the propsd sources from [the releases page][releases] and
+save it. These instructions assume you've saved the tarball to
+`/tmp/propsd.tar.gz`.
+
+Create a new folder for propsd. These instructions assume you're using
+`/opt` as that folder.
+
+~~~bash
+mkdir /opt
+~~~
+
+Use `npm` to install propsd.
```

Comment:
Good idea! Let's open an issue for that so we a) don't forget, and b) can have a larger conversation around it.